doc/release-notes.md

```diff
@@ -126,6 +126,30 @@ Updated RPCs
 New RPCs
 --------
 
+- Add `sweepwallet` RPC
+
+  The `sweepwallet` RPC spends some given UTXOs' complete balance to one or more
+  receivers without creating change.
+
+  By default, the `sweepwallet` RPC will empty the wallet completely leaving no
+  UTXOs behind. Optionally, the `send_max` option allows skipping uneconomic UTXOs
+  and therefore maximizing the received amount.
+
+  The `sweepwallet` RPC can process a combination of receiver addresses with and
+  without amounts specified. The specified amounts are paid first, and the
+  remainder split among the receivers with unspecified amounts. At least one
+  address must be provided without a specified amount to receive the balance left
+  after fees.
+
+  Instead of the complete UTXO pool, the call can be used to create a transaction
+  from a specific UTXO set. We recommend using `sweepwallet` to empty wallets
+  or to spend specific UTXOs in full.
+
+  The `sweepwallet` RPC therefore provides a less cumbersome way of spending
+  specific UTXOs or emptying wallets than subtract fee from output/amount (SFFO).
+  If the user wishes to specify a budget rather than a set of UTXOs to delimit a
+  transaction, they should continue to use SFFO. (#24118)
```

Comment:
I wonder if the release note isn't a bit long and whether the how-to info would be best documented elsewhere more permanently like in the sweepwallet help or in one of the doc/ files.

Reply:
I rewrote the release notes.